cmd/commands/common.go

```diff
@@ -215,21 +215,29 @@ func getRuntimeNameFromUserSelect(ctx context.Context) (string, error) {
 }
 
 func getRuntimeNameFromUserInput() (string, error) {
-	return getValueFromUserInput("Runtime name", "codefresh")
+	runtimeName, err := getValueFromUserInput("Runtime name", "codefresh", validateRuntimeName)
+	if err != nil {
+		return runtimeName, err
+	}
+	return runtimeName, err
```

Comment:
if you return `runtimeName, err` either way, you there is no need for the `if`.
you can just `return getValueFromUserInput....`